docs/pages/includes/disable-recording-cluster.mdx

```diff
@@ -0,0 +1,24 @@
+## Disabling session recording
+
+Session recording is enabled by default, and can be disabled in one of two ways:
```

Comment:
This no longer reads correctly. This text is specific to desktop recordings. I would revert.

Reply:
This still seems incorrect.

We say _can be disabled in one of two ways_ and then we list only one (not two) ways.

Disabling via RBAC is only supported for desktop, so I don't think we should use an include here.